code/datums/antagonists/datum_clockcult.dm

```diff
@@ -0,0 +1,133 @@
+//CLOCKCULT PROOF OF CONCEPT
+
+/datum/antagonist/clockcultist
+	prevented_antag_datum_type = /datum/antagonist/clockcultist
+	some_flufftext = null
+	var/datum/action/innate/hierophant/hierophant_network = new()
+
+/datum/antagonist/clockcultist/silent
+	silent_update = TRUE
+
+/datum/antagonist/clockcultist/Destroy()
+	qdel(hierophant_network)
+	..()
+
+/datum/antagonist/clockcultist/can_be_owned(mob/living/new_body)
+	. = ..()
+	if(.)
+		. = is_eligible_servant(new_body)
+
+/datum/antagonist/clockcultist/give_to_body(mob/living/new_body)
+	if(!silent_update)
+		if(iscarbon(new_body))
+			new_body << "<span class='heavy_brass'>Your mind is racing! Your body feels incredibly light! Your world glows a brilliant yellow! All at once it comes to you. Ratvar, the Clockwork \
+			Justiciar, lies in exile, derelict and forgotten in an unseen realm.</span>"
+		else if(issilicon(new_body))
+			new_body << "<span class='heavy_brass'>You are unable to compute this truth. Your vision glows a brilliant yellow, and all at once it comes to you. Ratvar, the Clockwork Justiciar, lies in \
+			exile, derelict and forgotten in an unseen realm.</span>"
+		else
+			new_body << "<span class='heavy_brass'>Your world glows a brilliant yellow! All at once it comes to you. Ratvar, the Clockwork Justiciar, lies in exile, derelict and forgotten in an unseen realm.</span>"
```

Comment:
Replace this with if(issilicon) first, in the else put this with "your mind is racing" text behind [iscarbon(newbody) ? "mindacing" : ""]